xdsl/interpreters/onnx.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, cast
+
+from xdsl.dialects import onnx
+from xdsl.interpreter import (
+    Interpreter,
+    InterpreterFunctions,
+    impl,
+    register_impls,
+)
+from xdsl.interpreters.shaped_array import ShapedArray
+
+try:
+    import numpy as np
+except (ImportError, ModuleNotFoundError) as exc:
+    print(exc)
```

Comment:
```suggestion
import numpy as np
```

Reply:
This will make some tests fail. Please use the same approach as we did for webgpu to work around them. You can search for "wgpu" in the project to see which parts of the code require special casing

Reply:
okay cool, but there isn't any special workaround that i see that "wgpu" does to handle this, ive done it now slightly differently  tho